app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -64,5 +65,17 @@ def stop_event_monitor_queue_on_credential_change
     def supports_cloud_tenants?
       false
     end
+
+    def sync_cloud_tenants_with_tenants
```

Comment:
* from where is this method called?
* And will it do more in the future? Right now it just syncs the root tenant.
* is the plural correct here or wouldnt it be `sync_cloud_tenant_with_tenant` ?